test/identityManager.js

```diff
@@ -145,6 +145,16 @@ contract('IdentityManager', (accounts) => {
       assert.isTrue(isOwner, 'user5 should be owner now')
     })
 
+    it('owner can not add owner that is already an owner', async function() {
+      try {
+        let tx = await identityManager.addOwner(proxy.address, user1, {from: user1})
```

Comment:
I don't think this should be user1 trying to add user1.
I know it's still an owner trying to add an existing owner, but this is a subset edge case of the condition being tested.